wallet/wallet.sol

```diff
@@ -363,16 +362,56 @@ contract Wallet is multisig, multiowned, daylimit {
             return true;
         }
     }
-    
+
+    // Gets the number of pending transactions
+    function numPendingTransactions() returns (uint) {
+        var pendingTransactionsCount = 0;
+        // Use m_pendingIndex.length to get all hashes, then count how many of the
+        // operations are transactions, because we don't want to store hashes twice
+        // and this is a local call anyway
+        for (uint i = 0; i < m_pendingIndex.length; i++) {
```

Comment:
`m_pendingIndex.length` is a dynamically-sized array. `i` would be a `uint8` (as per the [docs](http://solidity.readthedocs.io/en/latest/types.html#type-deduction).

If `m_pendingIndex.length > 255`, this is an endless loop.

I haven't looked through all the dapp, but can it be guaranteed that this won't end in always-OOG?